packages/puppeteer-core/src/cdp/Connection.ts

```diff
@@ -58,10 +54,6 @@ export class Connection extends EventEmitter<CDPSessionEvents> {
     this.#transport.onclose = this.#onClose.bind(this);
   }
 
-  static fromSession(session: CDPSession): Connection | undefined {
```

Comment:
This wasn't used anywhere, so I've decided to drop this trivial wrapper

Reply:
Acknowledged.

Reply:
Note that this method was part of the public API and we should not remove it to avoid breaking further clients.

Reply:
Reverted